homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -446,6 +555,46 @@ def native_value(self) -> int | None:
         return self.entity_description.value_fn(system_consumption)
 
 
+class EnvoyProductionPhaseEntity(EnvoySystemSensorEntity):
+    """Envoy phase production entity."""
+
+    entity_description: EnvoyProductionSensorEntityDescription
+
+    @property
+    def native_value(self) -> int | None:
+        """Return the state of the sensor."""
+        if self.entity_description.on_phase is None:
+            return None
+        if self.data.system_production_phases is None:
```

Comment:
Could combine these ifs since they do the same return

Reply:
Done